openlibrary/templates/type/author/view.html

```diff
@@ -129,13 +129,13 @@ <h2 class="collapse">
                 $if books.num_found > 1:
                     <img src="/images/icons/icon_sort.png" alt="Sorting by" width="9" height="11" style="margin-right:5px;"/> 
                     $if books.sort == 'editions':
-                        <strong class="lightgreen">Most Editions</strong> | <a href="$changequery(sort='old')#editions">First Published</a> | <a href="$changequery(sort='new')#editions">Most Recent</a> 
+                        <strong class="lightgreen">Most Editions</strong> | <a href="$changequery(sort='old')#editions">First Published</a> | <a href="$changequery(sort='new')#editions">Most Recent</a> | <a href="$changequery(sort='title')#editions">Alphabetically</a> 
```

Comment:
"Alphabetical" instead of "Alphabetically" works better both from the point of view of agreeing with the other options and being shorter.